src/client/locales/default.json

```diff
@@ -313,5 +313,5 @@
   "login_to_busy": "Login to Busy",
   "login_modal_description": "Login with your Steem account using SteemConnect to enjoy Busy at 100%",
   "login_with_steemconnect": "Login with SteemConnect",
-  "follows_you": "Follows you"
+  "follows_you": "Follows you",
```

Comment:
Technically it's not correct JSON with trailing comma, but I think Crowdin will work with this anyway.

Reply:
no you're right, I'll remove it

Reply:
let me remove it